docs/route.md

```diff
@@ -125,94 +125,46 @@ This is a simple property, so you can both read it and write to it.
 
 ##### m.route.Link
 
-This component can create a dynamic routable link:
+This component creates a dynamic routed link. Its essential function is to produce `a` links with local `href`s transformed to take account of the [route prefix](#mrouteprefix).
 
 ```javascript
-m(m.route.Link, {href: "/test"})
-```
-
-Using `m.route.Link` causes the link to behave as a router link - clicking it navigates to the route specified in `href`, instead of navigating away from the current page to the URL specified in `href`.
+m(m.route.Link, {href: "/foo"}, "foo")
 
-You can also set the `options` passed to `m.route.set` when the link is clicked by passing the `options` attribute:
-
-```javascript
-m(m.route.Link, {href: "/test", options: {replace: true}})
+// Renders to
+// <a href="#!/foo">foo</a>
 ```
 
-You can pass other attributes, too, and you can also specify the tag name used.
-
-```javascript
-m(m.route.Link, {
-	// Any hyperscript selector is valid here - it's literally passed as the
-	// first parameter to `m`.
-	selector: "span",
-	options: {replace: true},
-	params: {key: "value"},
-	href: "/test",
-	disabled: false,
-	class: "nav-link",
-	"data-foo": 1,
-	// and other attributes
-}, "link name")
-```
-
-Magic attributes used by this selector (except `href` and `disabled`) *are* removed while proxying, so you won't have an odd `selector="span"` or `options="[object Object]"` attribute show up in your link's DOM node. The above vnode renders to this hyperscript, assuming the prefix is the default `#!`:
-
-```javascript
-m("span", {
-	href: "#!/test",
-	onclick: function(e) {
-		// ...
-	},
-	disabled: false, // Only if you specify it
-	class: "nav-link",
-	"data-foo": 1,
-	// and other attributes
-})
-```
+Links accept a selection of special attributes: 
+* `selector` is what would be passed as the first argument to [`m`](hyperscript.md): any selector is valid, including non-`a` elements.
+* `params` & `options` are the arguments with the same names as defined in [`m.route.set`](#mrouteset).
+* `disabled`, if true, disables routing behaviour and any bound `onclick` handler, and attaches a `data-disabled="true"` attribute for accessibility hints; if the element is an `a`, the `href` is removed.
 
-You can also prevent navigation by, in an `onclick` handler, invoking `e.preventDefault()` or returning `false`. This is the same way you block other events, so it's pretty natural.
+*Routing behaviour cannot be prevented using the event handling API: use `disabled` instead.*
```

Comment:
Is that the case even for `m.route.Link`s with no custom selector?

Reply:
Yeah this is the crucial bug that motivated me to do this. Previous documentation said I could prevent default or return false in a click handler — you can’t. Instead I need to account for that logic at render time and it drastically changes the shape of the DOM 😫.

The previous docs were a rambling mess vaguely touching on all the weird caveats this behaviour engendered. I don’t believe we can fully account for this behaviour because it’s just so irrationally complex and undesirable. But if that major inaccuracy is accounted for, this is at least an improvement.